packages/typespec-test/test/todo_non_branded/generated/typespec-ts/src/api/todoItems/operations.ts

```diff
@@ -55,13 +55,7 @@ export function _$deleteSend(
   );
   return context
     .path(path)
-    .delete({
```

Comment:
Could you help to explain the code change in this project?
Are those changes expected?

Reply:
This is because of tcgc upgrade and expected and you could see comment here: https://github.com/Azure/autorest.typescript/pull/3407#discussion_r2244648094